Project.toml

```diff
@@ -25,7 +25,7 @@ TestSetExtensions = "98d24dd4-01ad-11ea-1b02-c9a08f80db04"
 AbstractAlgebra = "0.13, 0.18, 0.19, 0.20, 0.21, 0.22, 0.23, 0.24, 0.25, 0.26, 0.27"
 BenchmarkTools = "1"
 DataStructures = "0.18"
-Groebner = "0.2.4"
+Groebner = "0.2.8"
```

Comment:
Should we make this just "0.2" to simplify managing versions?

Reply:
Is `0.2` equivalent to `[0.2.0-0.3.0)` ?

Reply:
I think `0.2.8` (`[0.2.8, 0.3.0)`) is good enough, since versions `0.2.7` and before fail SI.jl tests

Reply:
Yes, I believe so.

Reply:
sounds good to me